android/KMEA/app/src/main/java/com/tavultesoft/kmea/KMManager.java

```diff
@@ -701,6 +722,17 @@ public static void switchToNextKeyboard(Context context) {
     if (InAppKeyboard != null) {
       InAppKeyboard.setKeyboard(pkgId, kbId, langId, kbName, langName, kFont, kOskFont);
     }
+
+    // Switch system keyboard to next IME so the user doesn't see
+    // SystemKeyboard.setKeyboard resetting to the first Keyman keyboard
+    if (index == 0 && switchToNextIME) {
+      InputMethodManager imm = (InputMethodManager) appContext.getSystemService(Context.INPUT_METHOD_SERVICE);
+
+      // TODO: this method is deprecated in API level 28
+      // Reference: https://developer.android.com/reference/android/view/inputmethod/InputMethodManager.html#switchToNextInputMethod(android.os.IBinder,%20boolean)
```

Comment:
If this is deprecated, can't we use the replacement function instead? [InputMethodService.switchToNextInputMethod](https://developer.android.com/reference/android/inputmethodservice/InputMethodService#switchToNextInputMethod(boolean))

Reply:
Currently we target API 27. I am investigating in another branch all the gradle dependencies to upgrade to API 28. It would involve updating the CI server too